gas_mock/HTTPResponse.js

```diff
@@ -1,14 +1,22 @@
 class HTTPResponse {
-  constructor (body) {
+  constructor(body) {
```

Comment:
Let's keep the current standard rule : https://eslint.org/docs/rules/space-after-function-name